_data/simple-icons.json

```diff
@@ -2609,6 +2609,11 @@
             "hex": "D33C43",
             "source": "https://cakephp.org/logos"
         },
+        {
+            "title": "Cal.com",
+            "hex": "292929",
+            "source": "https://cal.com/logo.svg"
```

Comment:
```suggestion
            "source": "https://design.cal.com/assets/logos"
```

Reply:
@uncenter I have just completed an update. Thank you so much again for your kind reply :)

Reply:
Awesome! I'm not sure if you know this but in the future you can click this "Commit suggestion" button instead of making the suggested changes separately.

![Screenshot 2024-05-08 at 20 01 51 (Arc)](https://github.com/simple-icons/simple-icons/assets/47499684/3a10bcb8-f174-44df-afe6-9e12618e4d54)

Reply:
Oh, I'll refer to it in the future :)